ext/afform/core/Civi/Afform/StatusChecks.php

```diff
@@ -28,8 +28,9 @@ public static function hook_civicrm_check($e) {
     if ($hasAuthx && $tokenFormCount && !in_array('jwt', \Civi::settings()->get('authx_auto_cred'))) {
       $e->messages[] = new \CRM_Utils_Check_Message(
         'afform_token_authx',
-        E::ts('Email token support has been configured for %1 form(s). This requires JWT authentication, <code>authx_auto_cred</code> does not include JWT. ', [
+        E::ts('Email token support has been configured for %1 form(s). This requires JSON Web Tokens are included as an acceptable credential for Auto Login in your AuthX configuration. Please <a href="%2">review your configuration here</a>.', [
```

Comment:
> This requires JSON Web Tokens are included

I had a hard time understanding this sentence. I think better would be "This requires _that_ JSON web tokens are ..."

Reply:
Thanks @Sjord that's helpful - I was definitely struggling to wordsmith it!